### DISCUSSION TITLE
Just wanted to say thank you

### DISCUSSION THREAD

**Discussion by jakguru** [bluenviron/mediamtx#2832] [General]
I wasn't sure where else to put this, but I wanted to say thank you for your hard work with this project. I've used it as a core component is my own open source project which helps people with Google Nest cameras get the streams in a way that allows them to be consumed by various applications such as Frigate and Home Assistant. While the working version of the application which includes your software is still in VERY early alpha, MediaMTX has proven to be consistent, stable and fully featured.

Thank you very much.

**Comment by aler9** [comment]
Hello, thank you very much for providing this feedback, the aim of this project has always be to create a stable component that can be used inside more complex projects and allows to abstract the complexity of handling media streams from the core features that are the main focus of most projects, from home surveillance to Machine learning-based analysis.

**Comment by dbason** [comment]
I also wanted to second this.  I know that working on an opensource project in your spare time can be pretty thankless, so thankyou.  This has been a great building block for a scalable media broker system I've been working on.

[a comment by an automated account is not shown]
